app/services/reports/deleted_user_accounts_report.rb

```diff
@@ -0,0 +1,40 @@
+require 'login_gov/hostdata'
+
+module Reports
+  class DeletedUserAccountsReport < BaseReport
+    REPORT_NAME = 'deleted-user-accounts-report'.freeze
+
+    def call
+      configs = JSON.parse(Figaro.env.deleted_user_accounts_report_configs || '[]')
+      configs.each do |report_hash|
+        name = report_hash['name']
+        emails = report_hash['emails']
+        issuers = report_hash['issuers']
+        report = deleted_user_accounts_data_for_issuers(issuers)
+        emails.each do |email|
+          UserMailer.deleted_user_accounts_report(
+            email: email,
+            name: name,
+            issuers: issuers,
+            data: report,
+          ).deliver_later
+        end
+      end
+    end
+
+    private
+
+    def deleted_user_accounts_data_for_issuers(issuers)
+      recs = []
+      issuers.each do |issuer|
+        transaction_with_timeout do
+          rows = DeletedAccountsReport.call(issuer, 10_000)
+          rows.each do |row|
+            recs << "#{row['last_authenticated_at']}, #{row['identity_uuid']}\r\n"
+          end
```

Comment:
What if we used the CSV class so we don't have to manually format these strings and join them?

```ruby
csv = CSV.new(String.new, col_sep: ', ', row_sep: "\r\n")
rows.each do |row|
  csv << [row['last_authenticated_at'], row['identity_uuid']
end
csv.string
```

Reply:
Related question, would it be useful if we generated the CSV as a file and sent it as an attachment? Or do the consumers want plaintext like this?

Reply:
> What if we used the CSV class so we don't have to manually format these strings and join them?
> 
> ```ruby
> csv = CSV.new(String.new, col_sep: ', ', row_sep: "\r\n")
> rows.each do |row|
>   csv << [row['last_authenticated_at'], row['identity_uuid']
> end
> csv.string
> ```

it's putting quotes around the date.  if you can tell me how to remove them I'll use the csv code.  i doubt the SP can read it this way.

Reply:
The reason it's adding quotes is because the timestamp has spaces in it, so there's no easy way to turn that off

Reply:
To bring this back: if we're checking with the partner, I'd love if we could verify that we can pass a "normal" CSV and not have to do spaces between elements, and allow the CSV generator to add quotes in as needed